internal/thumb/open_jpeg_test.go

```diff
@@ -17,8 +17,14 @@ func TestOpenJpeg(t *testing.T) {
 		}
 	})
 	t.Run("testdata/broken.jpg", func(t *testing.T) {
-		if _, err := OpenJpeg("testdata/broken.jpg", 0); err == nil {
-			t.Error("unexpected EOF while decoding error expected")
+		img, err := OpenJpeg("testdata/broken.jpg", 0)
```

Comment:
I see two possible test cases missing:
- When an image is broken in such a way that recovery is attempted but the recovery itself fails (line 82). This could be due to stat failing or mmap failing (that could actually be two different test cases, but it might be tricky to test separately)
- When an image is broken in such a way that recovery is done but opening it later fails (line 85)

Reply:
We do have metadata tests in the internal/meta package. The thumbs package is used by the photoprism package, which should also have resolution and orientation related tests.

Now, if we actively change/fix/update JPEG files, it makes sense to me that this requires additional testing beyond what currently exists?

Reply:
> We do have metadata tests in the internal/meta package. The thumbs package is used by the photoprism package, which should also have resolution and orientation related tests.

I see, thanks for pointing that out.

> Now, if we actively change/fix/update JPEG files, it makes sense to me that this requires additional testing beyond what currently exists?

Absolutely. I was just waiting to get indication that this approach is something you'd consider doing at all, but I'll take your comment as that now :)
Will try to add the tests as soon as I find some time.